### PR TITLE
[ZH] Translation of 06-3

### PR DESCRIPTION
**Pull request by lizruonan**
Might need to be fixed for some points.

### CODE REVIEW

**Comment by Atcold** [comment]
Thanks!

File: docs/zh/week06/06-3.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+translator: Elizabeth Zhao
+---
+
+## <!-- [Overview](https://www.youtube.com/watch?v=8cAffg2jaT0&t=21s) -->
+
+## [概述](https://www.youtube.com/watch?v=8cAffg2jaT0&t=21s)
+
+<!--RNN is one type of architecture that we can use to deal with sequences of data. What is a sequence? From the CNN lesson, we learned that a signal can be either 1D, 2D or 3D depending on the domain. The domain is defined by what you are mapping from and what you are mapping to. Handling sequential data is basically dealing with 1D data since the domain is just temporal inputs of X. Nevertheless, you can also use RNN to deal with 2D data, where you have two directions. -->
+
+循环神经网络（Recurrent Neural Networks, RNN）是一种可以用来处理数据序列的架构。在之前的CNN课程中，我们学到了信号可以是一维、二维、或者三维。其中维度取决于它的定义域（domain）。定义域是函数自变量的取值范围。因为序列数据的定义域是时轴(?)，所以处理序列数据是个1维的操作。不过，你也可以用RNN来处理双向的二维数据。
```
Comment (lizruonan) [inline docs/zh/week06/06-3.md:18]: Not sure how to translate "temporal inputs of X", but I put it as "时轴", which means "the time axis".

File: docs/zh/week06/06-3.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+</center>
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/basic_neural_net.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>图3:</b> 基本神经网络架构
+</center>
+
+<!--Yann's diagram adds these shapes between neurons to represent the mapping between one tensor and another(one vector to another). For example, in Figure 3, the input vector x will map through this additional item to the hidden representations h. This item is actually an affine transformation i.e. rotation plus distortion. Then through another transformation, we get from the hidden layer to the final output. Similarly, in the RNN diagram, you can have the same additional items between neurons.-->
+
+Yann在图示的网络神经之间添加了一些特殊图形，用来表示从一个张量到另一个张量的映射。如图3所示，输入向量x通过一个特殊图形映射到隐藏层h。这个特殊图形是一个放射变换（affine transformation），包括旋转加畸变（rotation and distortion）。经过另一个变换后，我们从隐藏层得到了最终输出。同理，在RNN图里的网络神经之间亦可添加同样的小图形。
```
Comment (lizruonan) [inline docs/zh/week06/06-3.md:48]: typo: 放射变换 --> 仿射变换

File: docs/zh/week06/06-3.md [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+### 问题
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/rnn_3.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>图20:</b> 梯度消失问题
+</center>
+
+<!--The figure above is a typical RNN architecture. In order to perform rotation over previous steps in RNN, we use matrices, which can be regarded as horizontal arrows in the model above. Since the matrices can change the size of outputs, if the determinant we select is larger than 1, the gradient will inflate over time and cause gradient explosion. Relatively speaking, if the eigenvalue we select is small across 0, the propagation process will shrink gradients and leads to the gradient vanishing.-->
+
+图20展示了一个经典RNN架构。我们用矩阵对RNN里的前一步进行旋转操作，在这个模型中用水平箭头表示。假设我们选择的行列式大于1，因为矩阵可以改变输出的大小，梯度会随时间增大并造成梯度爆炸。相对而言，如果我们选择的特征值小于0，则传播过程会收缩梯度并导致梯度消失。
```
Comment (lizruonan) [inline docs/zh/week06/06-3.md:275]: Just need to confirm: "the eigenvalue is small across 0" == "the eigenvalue is smaller than 0"?

Comment (Atcold) [reply docs/zh/week06/06-3.md:275]: Yeah, it should be "smaller than zero".

File: docs/zh/week06/06-3.md (hunk 2 above)
Comment (Atcold) [reply docs/zh/week06/06-3.md:48]: You can send a PR with this change only.

File: docs/zh/week06/06-3.md (hunk 1 above)
Comment (Atcold) [reply docs/zh/week06/06-3.md:18]: Feel free to ask on Slack for an opinion.

**Pull request merged**